01 - Community/People/Josh Plunkett.md

```diff
@@ -3,25 +3,58 @@ aliases:
 - Joshua Plunkett
 - Sklore
 - Sklex
-tags: 
-- 
+tags:
+- incubator
 publish: true
 ---
 
 # Josh Plunkett
 
-
+- GitHub: [Josh Plunkett](https://github.com/ObsidianTTRPGProject) ^github
 - Discord: `@sklore#8618` ^discord
+<!-- - Website: <https://> ^website-->
+<!-- - [[Publish sites|Publish site]]: <https://> ^publish -->
+
+%% Feel free to add a bio below this comment %%
+
 
 ## Author of
-Tutorials for using Obsidian as a TTRPG Campaign Manager. 
```

Comment:
Were these links moved somewhere else?

Reply:
They were removed at Josh's request. Confirming if that this should still be the case.

Reply:
Added the Publish site tutorial link, and added a one-liner bio. The videos were removed because most are "out of date" per his words, and he would rather the Publish site be pushed.